src/test/java/se/diabol/jenkins/pipeline/DeliveryPipelineViewTest.java

```diff
@@ -91,8 +90,8 @@ public void testOnJobRenamed() throws Exception {
         FreeStyleProject p1 = jenkins.createFreeStyleProject("build1");
 
         List<DeliveryPipelineView.ComponentSpec> componentSpecs = new ArrayList<DeliveryPipelineView.ComponentSpec>();
-        componentSpecs.add(new DeliveryPipelineView.ComponentSpec("comp1", "build1"));
-        componentSpecs.add(new DeliveryPipelineView.ComponentSpec("comp2", "build2"));
+        componentSpecs.add(new DeliveryPipelineView.ComponentSpec("comp1", "build1", null));
```

Comment:
I would like the test cases (DeliveryPipelineViewTest and GuiFunctionalIT) that use the ComponentSpec constructor to bind null to a constant "NONE", just to clarify the intention of the null usage in the test cases.